conditional-kotlin/src/main/kotlin/com/linecorp/conditional/kotlin/ConditionalExtension.kt

```diff
@@ -16,11 +16,104 @@
 
 package com.linecorp.conditional.kotlin
 
-import com.linecorp.conditional.ComposedCondition
-import com.linecorp.conditional.Condition
+import com.linecorp.conditional.*
+import java.util.concurrent.TimeUnit
 
-infix fun Condition.and(condition: Condition): ComposedCondition = this.and(condition)
+/**
+ * Returns a newly created [ComposedCondition].
+ * This [ComposedCondition] is composed with the AND operator.
+ *
+ * @param condition the [Condition] to compose.
+ *
+ * @throws NullPointerException if the [condition] is null.
+ */
```

Comment:
Like the existing code base, it seems that the code has become better by adding descriptions of functions through [KDoc](https://kotlinlang.org/docs/kotlin-doc.html#author) 👍